src/tunnel.c

```diff
@@ -176,7 +176,7 @@ static int pppd_run(struct tunnel *tunnel)
 				ofv_append_varr(&pppd_args, v[i]);
 		}
 
-		if (tunnel->config->pppd_use_peerdns)
+		if (tunnel->config->set_dns && tunnel->config->pppd_use_peerdns)
```

Comment:
maybe we should just change this to `if (tunnel->config->set_dns)` to address #349

Reply:
correcting myself: this would fix the current problem but leave us with a behavior that is even more confusing than it was before (it would send `usepeerdns` always, even when `pppd-use-peerdns=0` is configured),  [reverting](https://github.com/adrienverge/openfortivpn/issues/349#issuecomment-444470569) commit f383084 is probably the best solution